script/Arcueid/Day 10/10_00_ARC10_4.txt

```diff
@@ -1329,7 +1330,7 @@ Time passes, neither of us saying a word.
 [sha:cc1e926f8f78a4e303aef80c100a9db7f632cc24]{
 -- Page 388, Offset 25246.
 -- 「……ごめんなさい、志貴。
-"...I'm sorry, Shiki.
+"I'm sorry... Shiki.
```

Comment:
Might keep this one. Definitely think I'd avoid the ellipsis mid-sentence either way, the pacing feels off

Reply:
Had there been no voices, I'd've said that "Shiki, I'm... sorry." would be the preferrable alternative:

Once again, leading ellipsis is the enemy: I'd rather it be "I'm sorry, Shiki" at this point.

Reply:
No leading ellipsis and `sorry, shiki` fine with me